docs/en/datasets/recog.md

```diff
@@ -471,3 +476,289 @@ rm IIIT-CVid.zip
+
+  ```bash
+  mkdir imgur && cd imgur
+
+  git clone https://github.com/facebookresearch/IMGUR5K-Handwriting-Dataset.git
+
+  # Download images from imgur.com. This may take SEVERAL HOURS!
+  python ./IMGUR5K-Handwriting-Dataset/download_imgur5k.py --dataset_info_dir ./IMGUR5K-Handwriting-Dataset/dataset_info/ --output_dir ./imgs
+
+  # For annotations
+  mkdir annotations
+  mv ./IMGUR5K-Handwriting-Dataset/dataset_info/*.json annotations
+
+  rm -rf IMGUR5K-Handwriting-Dataset
+  ```
+
+- Step2: Generate `train_label.txt`, `val_label.txt` and `test_label.txt` and crop images with the following command:
+
+  ```bash
+  python tools/data/textrecog/imgur_converter.py PATH/TO/imgur
+  ```
+
+- After running the above codes, the directory structure should be as follows:
+
+  ```text
+  ├── imgur
+  │   ├── crops
+  │   ├── train_label.jsonl
+  │   ├── test_label.jsonl
+  │   └── val_label.jsonl
+  ```
+
+
+  ### FUNSD
+
+- Step1: Download [dataset.zip](https://guillaumejaume.github.io/FUNSD/dataset.zip) to `funsd/`.
+
+```bash
+mkdir funsd && cd funsd
+
+# Download FUNSD dataset
+wget https://guillaumejaume.github.io/FUNSD/dataset.zip
+unzip -q dataset.zip
+
+# For images
+mv dataset/training_data/images imgs && mv dataset/testing_data/images/* imgs/
+
+# For annotations
+mkdir annotations
+mv dataset/training_data/annotations annotations/training && mv dataset/testing_data/annotations annotations/test
+
+rm dataset.zip && rm -rf dataset
+```
+
+- Step2: Generate `train_label.txt` and `test_label.txt` and crop images using 4 processes with following command (add `--preserve-vertical` if you wish to preserve the images containing vertical texts):
+
+```bash
+python tools/data/textrecog/funsd_converter.py PATH/TO/funsd --nproc 4
+```
+
+### KAIST
+
+- Step1: Complete download [KAIST_all.zip](http://www.iapr-tc11.org/mediawiki/index.php/KAIST_Scene_Text_Database) to `kaist/`.
+
+  ```bash
+  mkdir kaist && cd kaist
+  mkdir imgs && mkdir annotations
+
+  # Download KAIST dataset
+  wget http://www.iapr-tc11.org/dataset/KAIST_SceneText/KAIST_all.zip
+  unzip -q KAIST_all.zip
+
+  rm KAIST_all.zip
+  ```
+
+- Step2: Extract zips:
+
+  ```bash
+  python tools/data/common/extract_kaist.py PATH/TO/kaist
+  ```
+
+- Step3: Generate `train_label.jsonl` and `val_label.jsonl` (optional) with following command:
+
+  ```bash
+  # Since KAIST does not provide an official split, you can split the dataset by adding --val-ratio 0.2
+  # Add --preserve-vertical to preserve vertical texts for training, otherwise
+  # vertical images will be filtered and stored in PATH/TO/kaist/ignores
+  python tools/data/textrecog/kaist_converter.py PATH/TO/kaist --nproc 4
+  ```
+
+- After running the above codes, the directory structure should be as follows:
+
+  ```text
+  ├── kaist
+  │   ├── crops
+  │   ├── ignores
+  │   ├── train_label.jsonl
+  │   └── val_label.jsonl (optional)
+  ```
+
+
+### ICDAR 2011 (Born-Digital Images)
```

Comment:
We can move the IC11&13 instructions to the top of this section, and mark the original IC13 instruction as deprecated

Reply:
done